django_pdb/testrunners.py

```diff
@@ -2,7 +2,7 @@
 
 from django.test.utils import get_runner
 
-from django.utils import unittest
```

Comment:
I think this would mean only supporting Python 2.7+.  See this note: https://docs.djangoproject.com/en/1.9/releases/1.7/#django-utils-unittest